src/Export.php

```diff
@@ -144,8 +152,12 @@ protected function write_insert_statements( $handle, $table_name ) {
 	 * @throws Exception
 	 */
 	protected function get_create_statement( $table_name ) {
-		$create = $this->translator->query( 'SHOW CREATE TABLE ' . $table_name );
-		return $create[0]->{'Create Table'} . "\n";
+		$table_name = $this->driver instanceof WP_SQLite_Driver
+			? $this->driver->get_connection()->quote_identifier( $table_name )
+			: $table_name;
+
+		$create = $this->driver->query( 'SHOW CREATE TABLE ' . $table_name );
+		return $create[0]->{'Create Table'} . ";\n";
```

Comment:
Should we `trim()` the produced query before we add a semicolon? In my tests, it ends up having a double semicolon at the end of each `CREATE TABLE` query.

Reply:
Fixed in https://github.com/Automattic/wp-cli-sqlite-command/pull/10/commits/8ee70fb63d5de90026848827576f6e54f2986060.